client/src/app/create/create.component.ts

```diff
@@ -24,7 +24,7 @@ export class CreateComponent implements OnInit {
 
     this.wordListService.wordList$
       .subscribe(results => {
-        const wordList = results.slice().reverse();
+        const wordList = [...results].reverse();
         this.wordList = wordList.map(item => Object.assign({}, item, {edit: false}));
```

Comment:
nie jestem po prostu przekonany, która z praktyk jest właściwa. Jeśli nie ma spread operator deafult, to nie rozszerzałbym specjalnie, choć w airnb proponują tak kopiowoać obiekty.